src/targets/cpu/include/migraphx/cpu/context.hpp

```diff
@@ -2,15 +2,65 @@
 #define MIGRAPHX_GUARD_RTGLIB_CONTEXT_HPP
 
 #include <migraphx/config.hpp>
+#include <migraphx/cpu/dnnl.hpp>
+#include <migraphx/cpu/parallel.hpp>
+#include <migraphx/par_for.hpp>
 
 namespace migraphx {
 inline namespace MIGRAPHX_INLINE_NS {
 namespace cpu {
 
+#ifdef USE_DNNL
 struct context
 {
     void finish() const {}
+
+    template <class F>
+    void bulk_execute(std::size_t n, std::size_t min_grain, F f)
+    {
+#if 1
```

Comment:
Why do we need a macro here?

Reply:
Yea thats debug code that can be removed.

Reply:
Fixed.